.github/workflows/containers.yml

```diff
@@ -23,7 +23,7 @@ jobs:
 
   # CeloCli standalone image
   celocli-standalone:
-    uses: celo-org/reusable-workflows/.github/workflows/container-cicd.yaml@v1.14.1
+    uses: celo-org/reusable-workflows/.github/workflows/container-cicd.yaml@v1.2.0
```

Comment:
this is an older version. is that correct

Reply:
Thanks!